world_positions: # measurements in meters
  names: ['C', 'M', 'A', 'B'] #, 'C1'] # F is on second row
  pos_x: [0.005, 0.005, 0.35, 0.35]
  pos_y: [-0.24, 0.24, 0.24, -0.24] # 0.57 and 0.01 is the midpoint that works for both grippers!
  pos_z: [-0.048,-0.048,-0.048, -0.048]
  
world_objects: # measurements in meters
  names: ['element', 'position', 'object', 'cube', 'base', 'roof']
  parents: ['', 'element', 'element', 'object', 'object', 'object']
  element: [0.001, 0.001, 0.001]
  position: [0.05, 0.05, 0.005]
  object: [0.001, 0.001, 0.001]
  base: [0.04, 0.11, 0.04]
  cube: [0.066, 0.066, 0.05]
  roof: [0.04, 0.067, 0.05]
  #pillar: [0.035, 0.035, 0.07]
  variance: 0.073